inference-engine/tests/functional/plugin/shared/include/behavior/test_model_repo.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include <string>
+
+std::string get_model_repo();
+
+namespace TestDataHelpers {
+
+    const char *getModelPathNonFatal() noexcept;
+
+    std::string get_data_path();
+
+    inline const char *getModelPathNonFatalDefault() noexcept {
+        if (const auto envVar = std::getenv("MODELS_PATH")) {
+            return envVar;
+        }
+
+#ifdef MODELS_PATH
```

Comment:
Why do you add `MODELS_PATH` from deprecated_tests? We refused to use these variables in new tests